docs/debugging/README.md

```diff
@@ -15,6 +15,11 @@ addition to being able to build PowerShell, you need:
 - .NET Core debugger installed (semi-automatic)
 - `powershell` executable in your path (self-host if not on Windows)
 
+The .NET CLI tools *must* be on your path for Visual Studio Code.
+`Start-PSBootstrap` installs the tools to `~/.dotnet`,
+but does not add this to your `PATH`.
+You can do this in Bash with `export PATH=$PATH:$HOME/.dotnet`.
```

Comment:
Possibly. @mirichmo can you test if that's the case, and if so, open another PR with further documentation?